docs/UserGuide.md

```diff
@@ -178,11 +178,15 @@ Adds a student to the address book.
 </box>
 
 <box type="tip" header="##### Tips">
+<markdown>
 
+* If your student's **NAME** contains special character, you should replace it with a suitable alternative.
```

Comment:
maybe

contains special characters, you should replace them with suitable alternatives.